docs-web/configuration/security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+We consider the security of the Ingress Controller paramount to the success of our users and use the following to 
+ensure the security of the Ingress Controller:
+* [Secure Development Life Cycle](https://www.microsoft.com/en-us/securityengineering/sdl/)
```

Comment:
would it make sense to clarify why we use the "Secure Development Life Cycle" or "FOSSA". For example, with FOSSA, is it for scanning dependencies?

Reply:
I'm not sure why we use Microsoft SDLC.
I think we use fossa for managing our 3rd party software, specifically vulnerability scanning and license conformance. @lucacome  ?

Reply:
yeah we use FOSSA for license scanning and Trivy for Docker images scanning

Reply:
awesome thanks